c_glib/arrow-glib/array.cpp

```diff
@@ -987,6 +1095,24 @@ garrow_int64_array_get_value(GArrowInt64Array *array,
   return static_cast<arrow::Int64Array *>(arrow_array.get())->Value(i);
 }
 
+/**
+ * garrow_int64_array_get_values:
+ * @array: A #GArrowInt64Array.
+ * @length: (out): The number of values.
+ *
+ * Returns: (array length=length): The raw values.
+ */
+const gint64 *
+garrow_int64_array_get_values(GArrowInt64Array *array,
+                              gint64 *length)
+{
+  auto arrow_array = garrow_array_get_raw(GARROW_ARRAY(array));
+  auto arrow_int64_array =
+    std::static_pointer_cast<arrow::Int64Array>(arrow_array);
+  *length = arrow_int64_array->length();
+  return reinterpret_cast<const gint64 *>(arrow_int64_array->raw_data());
```

Comment:
`gint64` is `long` but `int64` is `long long` on OS X (or clang?).

https://travis-ci.org/apache/arrow/jobs/246433948#L4032

```text
array.cpp:1113:10: error: static_cast from 'const value_type *' (aka 'const long long *') to 'const gint64 *' (aka 'const long *') is not allowed

  return static_cast<const gint64 *>(arrow_int64_array->raw_data());

         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

They are the same type on Linux (or GCC?).